packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx

```diff
@@ -127,11 +127,10 @@ There are two ways for icons to be added into the `@twilio-paste/icons` code pac
 1. Fork the Paste repository because PRs can only be open against forks and not branches for security reasons.
 2. Clone your newly forked Paste repository: `git clone <url>`
 3. Navigate inside the newly cloned repository with `cd paste`
-4. For each new icon you would like to create, add an `svg` icon file into the `packages/paste-icons/svg/` folder.
-5. From the root Paste folder, run the following command in your terminal: `yarn workspace @twilio-paste/icons convert-new`
-6. Verify your new icons were added correctly by checking the `packages/twilio-icons/src` folder to see the newly  created source files.
-7. At this point, run the following command to update our documentation with the new icons: `yarn workspace @twilio-paste/icons list-icons`
-8. Commit your changes and submit a new PR on Github!
+4. Add all the new `svg` icon files into the `packages/paste-icons/svg/` folder.
```

Comment:
I think it would be worth mentioning the naming convention for the exported assets. I don’t recall seeing anywhere that we expect a strict SnakeCase file name.

Reply:
That goes for the symbol in sketch when committed to abstract too

Reply:
@serifluous Do you know where I should add this in the instructions? Seems like it will happen on the design side primarily since we just drop the icons into the package and run a script blindly.

Reply:
Good point. I'll write a blurb to add to the "Setting up an icon in Paste Icons" section in a bit.

Reply:
1. Under the step "In the artboard names, replace "icon name" with the name of your icon.", add a callout that says:

**How to name icon artboards in Sketch**

Use camel case. For an interface icon, give the icon a name that describes its purpose rather than its content. For example, "24/edit" instead of "24/pencil".

If you're adding an icon used to brand a product, name it "product/[ProductName][SubproductName (optional)]". For example: "24/product/CodeExchangeCommunity".


2. Under the nested list of the step "Make sure the SVG code is clean:", add a list item (step 4) that says:

> If you're adding an icon used to brand a product, add the word "Product" to the beginning of the SVG file name. For example: "ProductCodeExchangeCommunity"

Reply:
Looks okay? https://paste-git-icons-simple-convert.twilio-dsys.now.sh/icon-system/how-to-add-an-icon @serifluous